api/src/main/java/org/openmrs/module/attachments/AttachmentsContext.java

```diff
@@ -345,6 +345,12 @@ public static double getCompressionRatio(double fileByteSize, double maxByteSize
 	 */
 	public static ContentFamily getContentFamily(String mimeType) {
 		ContentFamily contentFamily = ContentFamily.OTHER;
+		if (StringUtils.equals(mimeType, "some type")) {
```

Comment:
Unfortunately i haven't been able to get the correct MIME type for all executables.  
 i think that i could use `application/octet-stream` according to https://www.lifewire.com/file-extensions-and-mime-types-3469109 but i think it isn't right basing on the [iana media types](https://www.iana.org/assignments/media-types/media-types.xhtml#Ozgur_Oyman)

Reply:
@jnsereko maybe take it the other way around. Think of a couple of obviously insecure file types (.js, .exe, .sh, .sql, ... etc) and see what the MIME types are for those guys, and start building an exclusion list that way.
You could probably start with:
* `application/octet-stream`
* `text/javascript` (and its cousins, see [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#textjavascript)). 

@isears @ibacher?

Reply:
Yes, just denying the upload of `aplication/octet-stream` and `text/javascript` would be a big improvement over what we have now (which is no filtering).

Building that initial functionality is the difficult part. Once we have it for one or two mime-types we can always extend it to include others if we need to.